src/Document/TextElement.ts

```diff
@@ -9,16 +9,39 @@ import {
 } from '../util';
 import Font from '../Font';
 import BoundingBox from '../BoundingBox';
+import Property from '../Property';
 import Document from './Document';
 import Element from './Element';
 import FontElement from './FontElement';
 import GlyphElement from './GlyphElement';
 import RenderedElement from './RenderedElement';
 
 export default class TextElement extends RenderedElement {
+	private static inheritAttribute(child: TextElement, name: string): string | null {
+		function isFirstChild(child: TextElement): boolean {
```

Comment:
Let's move it to [Element](https://github.com/canvg/canvg/blob/master/src/Document/Element.ts)

```js
child.isFirstChild()
```